Makefile

```diff
@@ -140,6 +152,22 @@ mock-gen-clean:
 docs-uml:
 	@go-plantuml generate --recursive --directories cmd --directories internal --directories pkg --out $(DOCS_DIR)/$(UML_OUT_FILE)
 
+.PHONY: test-integration-ci ## run integration tests on ci (github actions)
+test-integration-ci:
+	@METRO_TEST_HOST=metro-producer go test ./... -tags=integration,musl
```

Comment:
does this mean these tests are being run only for producer component ?

Reply:
Integration tests run against API endpoints which is being exposed by producer today. I am thinking we can rename it as `web` and have both producer and pull consumer APIs be part of it?